tests/LocalCacheTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
```

Comment:
Can you also remove the PhpStorm default header?

Reply:
removed